### ISSUE TITLE
tli doesn't map ' to a proper IPA symbol

### ISSUE THREAD

**Issue by joanise** [roedoejet/g2p#81]
Discovered while moving the tokenizer to g2p:

`tli` does not map to `'` to a proper IPA symbol:
```
$ g2p convert  "k'w" tli tli-ipa
k'ʷ
$ g2p convert  "k'w" tli eng-arpabet
K 'W
```

Additional side effect: the tokenizer for `tli` will not recognize `'` as being part of the word.

Question: is it correct to map it to the glottal stop, like in other west coast languages:

    ',ʔ

**Comment by joanise** [comment]
Reading up on the Tlingit orthography on Wikipedia (https://en.wikipedia.org/wiki/Tlingit_alphabet), it appears `ʼ` (\u02BC) would be more appropriate, the ejective stop character (https://en.wikipedia.org/wiki/Dental_and_alveolar_ejective_stops).
Panphon does not know it, though, so I'm guessing in tli-ipa -> eng-ipa, we should manually map \u02BC to the full glottal stop, so that we get `HH` in eng-arpabet.

**Comment by roedoejet** [comment]
Hi @joanise , thanks for this. Yes, you're right, it marks ejectives - frustratingly, I believe periods mark glottal stops. Ejectives should get mapped to their plain unvoiced stop equivalents, so `k\u02BC` should go to `k`. Panphon does recognize the character, but will just remove it in the generate-mapping step:

```
>>> import panphon.distance
>>> dst = panphon.distance.Distance()
>>> dst.weighted_feature_edit_distance("kʼ", "k")
0.125
```

I suppose the thing to do would be either to replace all `'` with `\u02BC`, or to add individual rules for each ejective (if we thought `'` might be used legitimately in punctuation). I think the individual rules option might be the best.

**Comment by joanise** [comment]
Ah, I see! thanks for the explanations. We need to have ' somewhere in the g2p, because otherwise the tokenizer will not consider it a word character. `g2p generate-mapping` does not like having \u02BC in a rule on its own, though. I think a reasonable solution is to have a rule for each ejective plosive, as you suggest, in tli-ipa -> eng-ipa.

As for the tokenizer, maybe the simplest solution is to have the no-op rule `','` in `tli_equiv.csv` mapping `'` to itself. Then the tokenizer would include the ASCII `'` in the alphabet it's looking for.

**Comment by joanise** [comment]
I take it back, `g2p generate-mapping` was quite happy to generate this rule:
```
    {
        "in": "\u02bc",
        "out": "",
        "context_before": "",
        "context_after": ""
    }
```
and maybe that's just the simple solution we should implement, and add the rule(s) mapping `'` to `\u02BC` as appropriate in `tli_to_ipa.csv`.

**Comment by joanise** [comment]
About the dot mapping to the glottal stop, how does the language mark it's end of sentence punctuation? Is there ambiguity?

**Comment by joanise** [comment]
Next question, can the glottal stop occur at the end of a word? I could disambiguate by saying only map the . if it's not word-final.

**Comment by joanise** [comment]
Fixed by PR #82
